jormungandr-lib/Cargo.toml

```diff
@@ -30,7 +30,7 @@ quickcheck = "0.9"
 chain-impl-mockchain = { path = "../chain-deps/chain-impl-mockchain", features = [ "property-test-api" ] }
 chain-addr      = { path = "../chain-deps/chain-addr", features = [ "property-test-api" ] }
 chain-crypto    = { path = "../chain-deps/chain-crypto", features = [ "property-test-api" ] }
-chain-core    = { path = "../chain-deps/chain-core", features = [ "property-test-api" ] }
+chain-core    = { path = "../chain-deps/chain-core"}
```

Comment:
The missing space before the closing brace bugs me, here and below :grin: Otherwise looks good.

Reply:
It bugs me too actually 🤦